en/advanced/system_startup.md

```diff
@@ -47,3 +47,17 @@ The `extras.txt` can be used to start additional applications after the main sys
 ### Starting a custom mixer
 
 By default the system loads the mixer from `/etc/mixers`. If a file with the same name exists in `/fs/microsd/etc/mixers` this file will be loaded instead. This allows to customize the mixer file without the need to recompile the Firmware.
+#### Example
+The following example shows how to add a custom aux mixer:
+  * Create a file on the SD card, `etc/mixers/gimbal.aux.mix` with your mixer
+	content.
+  * Then to use it, create an additional file `etc/config.txt` with this content:
+```
```

Comment:
Just FYI, you can indent this to the level of "Then to ..." above to put this within the preceding bullet point.